src/utils/getTimePeriods.ts

```diff
@@ -10,7 +10,7 @@ const YEAR_IN_SECONDS = 31557600
  * @param seconds
  */
 const getTimePeriods = (seconds: number) => {
-  let delta = seconds
+  let delta = Math.abs(seconds)
```

Comment:
This change might have repercussions elsewhere but my thinking was it should not matter if the number is negative or positive to get the time periods.